nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-processors/src/main/java/org/apache/nifi/processors/influxdb/ExecuteInfluxDBQuery.java

```diff
@@ -189,9 +209,10 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
 
         try {
             long startTimeMillis = System.currentTimeMillis();
-            QueryResult result = executeQuery(context, database, query, queryResultTimeunit);
+            int chunkSize = Integer.valueOf(context.getProperty(INFLUX_DB_QUERY_CHUNK_SIZE).evaluateAttributeExpressions(outgoingFlowFile).getValue());
```

Comment:
You can use `asInteger()` or `asLong()` here instead of using `Integer.valueOf` and `getValue()`